distributed/tensor_parallelism/requirements.txt

```diff
@@ -3,4 +3,4 @@
 --pre
 --extra-index-url https://download.pytorch.org/whl/nightly/cu118
 --extra-index-url https://download.pytorch.org/whl/nightly/cu121
-torch >= 2.2.0.dev0; sys_platform == "linux"
+torch >= 2.3.0.dev0; sys_platform == "linux"
```

Comment:
btw would this example work on T4 GPUs? Most of the distributed work here is still untested but there's no real reason for that IMO

Setting up distributed CI is as simple as `runs-on: 4-core-ubuntu-gpu-t4`

Reply:
Cool let me try this!